content/20.treatments.md

```diff
@@ -86,6 +86,68 @@ A list of current studies and their results, using carefully the information req
 
 Summarize the state of the antiviral approach.
 
+##### Hydroxychloroquine and Chloroquine
+
+Hydroxychloroquine (HCQ) and chloroquine (CQ) are quinolones that have been widely used as anti-malarial and autoimmune drugs in humans.
+HCQ and CQ act through the same mechanism of action, but HCQ has a better safety profile [@doi:10.1136/ard.2008.101766].
+HCQ/CQ have reported to have anti-viral effects, including in HIV-1, SARS-CoV, MERS-CoV, and SARS-CoV-2 [@doi:10.1016/S1473-3099(06)70361-9; @doi:10.1128/AAC.03011-14; @doi:10.1093/cid/ciaa237; @doi:10.1038/s41422-020-0282-0].
+In vitro studies of CQ use for SARs-CoV infection suggest anti-viral activity is due to elevation of endosomal pH and terminal glycosylation of angiotensin-converting enzyme 2, the cellular entry receptor [@doi: 10.1186/1743-422X-2-69].
+In vitro studies of SARS-CoV-2 infection show both HCQ and CQ to be effective, with HCQ being more potent [@doi:10.1093/cid/ciaa237].
+Thus, there is much excitement about HCQ as a potential therapeutic for SARS-CoV-2 and COVID-19.
+This section will discuss current efforts on assessing the potential role of HCQ in treating COVID-19.
+
+###### Current Evidence
```

Comment:
Hey Feng: really great work on this section, it provides a clear and effective critique of current hydroxychloroquine data. I think there are a few more trials floating around and I'll try to collect them here. 

This is one of them: Jun et al, Jrnl Zhejiang Univ. http://www.zjujournals.com/med/EN/10.3785/j.issn.1008-9292.2020.03.03

My takeaways: open-label RCT of 30 pts. No difference in viral clearance at 7d (87% HCQ vs 93% usual care), p>0.05. Big limitations: small study (1 center), surrogate endpoint, underpowered... interpret w/ caution

Reply:
I cited this particular one! It's the study where only the abstract is available in English. If you come across other clinical trials on HCQ (or otherwise - I know of at least one on Lopinavir-Ritonavir), I'd love if you posted here. I'm sure I'm missing some important ones.